src/main/java/io/vertx/httpproxy/HttpProxy.java

```diff
@@ -80,6 +80,12 @@ default HttpProxy origin(int port, String host) {
   @Fluent
   HttpProxy originSelector(Function<HttpServerRequest, Future<SocketAddress>> selector);
 
+  @Fluent
+  HttpProxy onProxyRequestCompletion(Handler<Future<Void>> handler);
```

Comment:
that should be `Handler<Void>`

Reply:
I went with the suggestion of `Handler<Event>`